### DISCUSSION TITLE
CME Widi master

### DISCUSSION THREAD

**Discussion by alros100** [Builty/TonexOneController#44] [Q&A]
Hi

Does anyone have these widi master Bluetooth midi interconnect[ Toggle](https://www.cme-pro.com/widi-master/)

I got two to test out and they connect to each other very well by BT. They see the tonexBT device set to peripheral but they can't connect

Wondering if anyone had succes with them before I send them back to Amazon 

Thanks
Al

**Comment by Builty** [comment]
I don't recall anyone reporting anything about these. 
I don;t understand why they wouldn't connect. The tonexBT peripheral is just a basic Midi device. Does the WIDI master need any special configuration to be paired directly to a Midi device?

**Reply by alros100** [reply]
Thanks for the response and happy new year !

The documentation says it's all automatic find and connect.  But to connect more than one device, it has an app to connect and group them. I'll factory reset and try it one more time with just the ToneX one

Thanks again
Al

**Reply by alros100** [reply]
Builty,

I retried and it does connect and work if I only have one of these widi masters powered. Just wanted to post back as your device does work as expected.

If I power on the second widi, they won't connect to the ToneXone. They do have a group mode but I can't get that to connect to ToneX one - must be a proprietary thing to get us to buy more widi 

Thanks again and happy new year
Al

**Reply by Builty** [reply]
Thanks. Happy New Year to you also.
I did read the Widi manual briefly. It does say that to connect to a BT peripheral, to use only one of the Widi master devices.
I guess if you power up both of the Widi's then they connect to each other instead.

**Reply by alros100** [reply]
Thanks Builty

So  believe it or not their CEO responded to my question! it turns out you can't use their app for this situation. You have to put one of the widi devices into an automatic group mode. Then you turn on all the non widi and widi peripherals and it creates a pairing group.

It works perfectly now if anyone needs to expand to Bluetooth instead of cables. Just wasn't initially obvious. But in truth their website had a blog post explaining this.

Currently my FM9 has the master wifi, big ToneX has another widi and the Builty ToneX one is the second peripheral. Their system can group a master and four peripherals

Thanks again !
Alaa
